### ISSUE TITLE
API_64BIT_LOGIN_APP_OUTDATED_286

### ISSUE THREAD

**Issue by juniorkalvin7** [vysheng/tg#1733]
phone number: +XXXXXXXXXX
 *** 1638565210.570832 Notification API_64BIT_LOGIN_APP_OUTDATED_286: You are using an outdated app that is no longer supported. To access your messages, please update your app to the latest version.
 *** 1638565210.571147 Incorrect phone number
phone number: SIGTERM/SIGINT received
halt

**Comment by andrew-phi** [comment]
Confirmed. They've forcibly outdated older clients, including desktop 1.x 2.x.

**Comment by varungup** [comment]
Yes, they are not sending any updates. However, if you are already authorized, then response is coming for the commands.

**Comment by sijumanuel** [comment]
same error

**Comment by abitmore** [comment]
Same issue reported here: https://github.com/kenorb-contrib/tg/issues/51
